src/history/index.js

```diff
@@ -1,3 +1,31 @@
-import {history} from 'backbone';
+let _navigate, _back;
 
-export default history;
+// Define the navigation functions depending on backbone or react-router
+export const setNavigationFunctions = (navigate, back) => {
+    _navigate = navigate;
+    _back = back;
+    window._navigate = navigate;
```

Comment:
It seems you have a global problem and you try to solve it  with `window` which is wrong. We will try to find a better way to do this.